### PR TITLE
change html

### PR DESCRIPTION
**Pull request by salaos**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+< lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>NINA SAALFRANK</title>
+  </head>
+
+
+  <!-- HERE WE GO-->
```
Comment (alinawinter) [inline index.html:11]: Very cool and helpful with the comments!

File: index.html [hunk 2]
```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+< lang="en">
```
Comment (alinawinter) [inline index.html]: missing "html": <html lang="en">

File: index.html [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+        <a href="https://de.linkedin.com/">LINKEDIN</a>
+        <a href="https://figma.com/">FIGMA</a>
+        <a href="https://instagram.com/">INSTAGRAM</a>
+        <a href="https://github.com/salao">GITHUB</a>
+      </section>
+    </main>
+  </body>
+
+  <footer>
+    <!--FOOTER-->
```
Comment (alinawinter) [inline index.html]: maybe this comment better in line 103?

File: index.html [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+    <section id="imprint">
+      <br />
```
Comment (alinawinter) [inline index.html:109]: Maybe this part is also better to have 2 lines before?

File: index.html [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+          height="200px"
+        />
+
+        <img
+          src="https://images.pexels.com/photos/15394523/pexels-photo-15394523.jpeg?auto=compress&cs=tinysrgb&w=800&lazy=load"
+          alt="photo2"
+          height="200px"
+        />
+
+        <p><a href="https://www.google.com">&#187; see more</a></p>
```
Comment (alinawinter) [inline index.html:89]: I think the "see more" should go between <a> and </a> or is this something we haven't learned so far?

**Review by alinawinter — CHANGES_REQUESTED** [review]
Really nice work, already a nice design

File: index.html (hunk 3 above)
Comment (salaos) [reply index.html]: true, thanks!

File: index.html (hunk 2 above)
Comment (salaos) [reply index.html]: ups danke

**Pull request merged**